src/hdf5_hl.ts

```diff
@@ -529,27 +534,27 @@ export class Attribute {
     this.shape = metadata.shape;
   }
 
-  get value() {
+  get value(): OutputData | null {
```

Comment:
Just being a bit more explicit (here and on a few other methods). It helps double check that the implementation of the method is typed correctly (notably `get_attr` in this case).